website/source/docs/agent/configuration/telemetry.html.md

```diff
@@ -58,6 +58,14 @@ The following options are available on all telemetry configurations.
 - `publish_node_metrics` `(bool: false)` - Specifies if Nomad should publish
   runtime metrics of nodes.
 
+- `backwards_compatible_metrics` `(bool: false)` - Specifies if Nomad should
+  publish metrics that are backwards compatible with versions below 0.7, as
+  post version 0.7, Nomad emits tagged metrics.
+
+- `disable_tagged_metrics` `(bool: false)` - Specifies if Nomad should not emit
```

Comment:
I would document that new metrics will only be added to tagged metrics. Further that these options are used to transition monitoring to tagged metrics and will eventually be deprecated.